src/Components/CameraControl.jsx

```diff
@@ -128,13 +139,15 @@ export function hasValidUrlParams() {
 
 /**
  * Adds camera coords to url.
- * @param {Object} viewer IFCjs viewer that contains a camera.
+ * @param {Object} cameraControls obtained from the viewer
  */
-export function addCameraUrlParams(viewer) {
-  const camera = viewer.IFC.context.ifcCamera
-  const position = camera.cameraControls.getPosition()
+export function addCameraUrlParams(cameraControls) {
+  if (!cameraControls ) {
```

Comment:
This seems like an error.  Is there a valid case where this method is called with null/undefined?  If so pls document if not, throw error